Apps/Src/UpdateVelocity.c

```diff
@@ -28,7 +35,7 @@ static float convertPedaltoMotorPercent(uint8_t pedalPercentage)
     return pedalToPercent[pedalPercentage];
 }
 
-// Convert a float velocity to a desired RPM
+// Convert a float velocity in meters per second to a desired RPM
 static float velocity_to_rpm(float velocity) {
     float velocity_mpm = velocity * 60.0f; // velocity in meters per minute
```

Comment:
Comment says meters per second, conversion is done on meters per minute?

Reply:
It takes in a meters per second value, multiplies by sixty to generate an MPM value, and then determines rpm by doing that value divided by the circumference.